demo/admin/src/news/generated/NewsForm.tsx

```diff
@@ -0,0 +1,226 @@
+// This file has been generated by comet admin-generator.
+// You may choose to use this file as scaffold by moving this file out of generated folder and removing this comment.
```

Comment:
How exactly is the flow if I want to use the components as a scaffold? If I remove a file from the generated folder, the CRUD generator just regenerates it the next time.

- Do I have to remove the entity from the generator config if I want to use it as a scaffold?
- Does this mean I can either auto-generate or scaffold all components of an entity (no mixture)?

- Do I have to remove the entity from the generator config?
- What would be the best way to just change one component (e.g. the form) but keep the other components generated? Or is it either all components auto-generated or all scaffolded?

Reply:
In practice I think you will remove it from the generator config and use all files as scaffolding. (no mixture)

(in theory you could have eg. grid scaffolded and grid generated)

Generating different, smaller parts that you can pick from would be a different approach I decided against, as that would be no good scaffolding (and especially in admin scaffolding is the most common use case)